tests/testthat/test-modspec.R

```diff
@@ -113,7 +113,7 @@ test_that("multiple blocks allowed or not allowed", {
   )
   expect_is(mod, "mrgmod")
   
-  code <- "$ODE a = 3\n$ODE b = 55\n$PARAM x = 3"
+  code <- "$CMT a\n$ODE dxdt_a = 3\n$ODE b = 55\n$PARAM x = 3"
```

Comment:
I made this change to the test, but looking at it again, wondering why it's popping up.

Reply:
```r
> code <- "$ODE a = 3\n$ODE b = 55\n$PARAM x = 3"
> mcode("foo", code)
Warning: Could not find a $INIT or $CMT block.
error.

```

Reply:
Ok; this is fallout from the `warn` / `quiet` business. We make the tests run with `quiet`, so this warning was suppressed before, when `quiet` could also suppress `warn`.  I think the current behavior is correct ... notify user no matter what if there is `$ODE` code but no compartments.  Maybe should be an error; but it's been a warning that the user would normally see from the start.